usr/share/rear/backup/NETFS/default/41_save_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# save all found capapilities to file 
+
+if [ "$BACKUP_CAP" == "y" ] ; then
+	Log "Saving capabilities to File." 
+	if which getcap >/dev/null 2>&1 ; then
+		getcap -r / 2>/dev/null | grep -v $ISO_DIR > $VAR_DIR/recovery/capabilities
```

Comment:
IMHO this code should reflect the actual paths that will be part of the backup. If you look at `backup/NETFS/default/50_make_backup.sh` you will see how NETFS backup works.

The include and exclude list of backup paths is provided in `$TMP_DIR/backup-include.txt` and `$TMP_DIR/backup-exclude.txt`

On my Ubuntu desktop `getcap -r /` throws a lot of errors for `/proc`, `/run` etc. which you ignore ATM. I think if you run `getcap` only on the relevant files and directories then you don't need to ignore the errors but can add them to the logfile. In case of a real problem this will help the user to understand why his recovered system does not work properly.

What is the size of the capabilities file? Will it make the boot image too large? Maybe compress with gzip or bzip2 or even put it in OUTPUT_URL instead of on the rescue media. After all, if the backup is not available then the capabilities file is also not needed.

Last, on my desktop with an SSD the `getcap -r /`  takes 3 seconds on the repeated runs and finds 12 items. On a server this might take much longer especially since in most cases the caches are not yet primed. I think that adding a progress bar would be nice to show the user what takes so long.